src/_guides/language/sound-problems.md

```diff
@@ -371,25 +371,25 @@ in a compile-time error.
 {:.fails-sa}
 <?code-excerpt "strong/lib/common_problems_analysis.dart (func-dynamic)" replace="/String/[!$&!]/g"?>
 {% prettify dart %}
-typedef bool Filter(dynamic any);
+typedef Filter = bool Function(dynamic any);
 Filter filter = ([!String!] x) => x.contains('Hello');
 {% endprettify %}
 
 {:.console-output}
 <?code-excerpt "strong/analyzer-2-results.txt" retain="/type '\(String\) → bool'.*common_problems/"?>
 ```nocode
-error • A value of type '(String) → bool' can't be assigned to a variable of type '(dynamic) → bool' • invalid_assignment
 error • The function expression type '(String) → bool' isn't of type '(dynamic) → bool'. This means its parameter or return type does not match what is expected. Consider changing parameter type(s) or the returned type(s) • strong_mode_invalid_cast_function_expr
+error • A value of type '(String) → bool' can't be assigned to a variable of type '(dynamic) → bool' • invalid_assignment
 ```
 
-#### Fix: Add generic type parameters _or_ cast from dynamic explicitly
+#### Fix: Add type parameters _or_ cast from dynamic explicitly
```

Comment:
@munificent - are you ok with this change? I dropped "generic", since, under the new syntax

```dart
typedef Filter<T> = bool Function(T any);
```

the`<T>` in `Filter<T>` type isn't a _generic type_ (although it is a type parameter).

Reply:
👍